### ISSUE TITLE
OTG USB support

### ISSUE THREAD

**Issue by dbogdanov** [timusus/Shuttle#475]
It is not clear from the documentation if playing music from OTG USB is supported. Can this be explicitly stated in the documentation?

**Comment by dbogdanov** [comment]
After a quick test, it looks like there is no support.

**Comment by jMavarez** [comment]
OTG support it's dependent on the device sometimes, does your device support it?

**Comment by dbogdanov** [comment]
Yes, it is Google Pixel. However, I could not access the USB from the app while Foobar worked no problem.

**Comment by jMavarez** [comment]
Interesting, I'll with my pixel when I get home, but I think I know how to solve this.
